Playbooks/Linux setup.yml

```diff
@@ -44,17 +38,17 @@
       apt:
         update_cache: yes
         upgrade: 'yes'
-        
+
 - name: Install oh-my-zsh
   hosts: "*"
   roles:
     - role: gantsign.oh-my-zsh
       users:
         - username: master ## Username of the account you'd like to install Oh-My-Zsh to. Use "ansible-galaxy install gantsign.oh-my-zsh" first.
-                         ## https://galaxy.ansible.com/gantsign/oh-my-zsh  
+                         ## https://galaxy.ansible.com/gantsign/oh-my-zsh
```

Comment:
I added a requirements.yaml file, that should make these `ansible-galaxy install` comments obsolete.